roseau/load_flow/models/lines/parameters.py

```diff
@@ -816,7 +919,7 @@ def from_power_factory(  # noqa: C901
         cohl: Literal[0, "Cable", 1, "OHL"] = "Cable",
         conductor: Literal["Al", "Cu", "Ad", "As", "Ds"] | None = None,
         insulation: Literal[0, "PVC", 1, "XLPE", 2, "Mineral", 3, "Paper", 4, "EPR"] | None = None,
-        section: float | Q_[float] | None = None,
+        section: FloatScalarOrArrayLike1D | None = None,
```

Comment:
Power factory has a scalar section, this should not change. 
```suggestion
        section: float | Q_[float] | None = None,
```